atmos_validation/schemas/metadata.py

```diff
@@ -48,7 +48,9 @@ class HindcastMetadata(CommonMetadata):
 class MeasurementMetadata(CommonMetadata):
     """Extra global attributes required if data_type == "Measurement"."""
 
+    asset: Optional[str]
```

Comment:
Do these new fields also need to be aded in partial_matadata.py as well?

Reply:
When the changes are incorporated in the internal repo, we will need to update that to pass tests, yes